nix/haskell.nix

```diff
@@ -85,6 +85,24 @@ let
             '';
           };
 
+
+        # TODO: How do we bundle the config in the derivation, and
+        # do we even want to do this? Maybe there are better alternatives.
+        packages.cardano-wallet-byron.components.benchmarks.restore = {
+           build-tools = [ pkgs.makeWrapper ];
+           preBuild = ''
+             cat ${src + /lib/byron/test/data/bench-mainnet-configuration.yaml}
+             ls $out
+             cp ${src + /lib/byron/test/data/bench-mainnet-configuration.yaml} $out/config.yaml
```

Comment:
We — probably — need a custom config. 

1. To stop the node trying to write logs the the working directory where it doesn't have permissions
2. To disable the node's LiveView GUI

I suspect something like this could be done, but this in particular didn't work

Reply:
@rvl Does bundling the config like this sound like a sane idea to begin with, and do you know how to? (I suspect it's simple)

Reply:
We could include our own logging config, but in this case I don't think we need to.

The liveview stuff probably won't affect the benchmark so it can stay. The other logging settings provided by cardano-node might help with debugging the benchmark.

The "trick" I used in cardano-launcher is to change working directory to somewhere where cardano-node is permitted to write its logs. So remove the `--run "cd $src"` line from below, because that forces the cwd to be a read-only location.